website/src/css/custom.scss

```diff
@@ -108,10 +109,6 @@ html[data-theme='dark'] {
   --ifm-link-color: #087EA4;
 }
 
-.navbar {
-  background-color: var(--ifm-background-color);
```

Comment:
If you set `  --ifm-navbar-background-color:` in `[data-theme='dark'] {`, we should be good to go.

Reply:
I'll fix it

Reply:
5a0a21b55348136007c87bb7a0dd7bff65b65b66